app/package.json

```diff
@@ -49,19 +49,14 @@
     "express": "~4.18.3",
     "formik": "~2.4.5",
     "keycloak-js": "~23.0.7",
-    "leaflet": "~1.9.4",
-    "leaflet-draw": "~1.0.4",
-    "leaflet-fullscreen": "~1.0.2",
-    "leaflet.locatecontrol": "^0.76.1",
+    "leaflet": "^1.9.4",
```

Comment:
leaflet still installed, any requirements still needing it?

Reply:
Just Core leaflet. It's using it for a calculation I haven't groked yet.

Reply:
Mind you... I thought I got rid of the plugins there... Will check .

Reply:
Hmmm... I don't see those plugins on my version. :thinking:

Reply:
Oh never mind... They were removed.

Reply:
👍

Reply:
Looking at Leaflets source code... Think I can bring that logic internal.... and remove leaflet entirely.

Reply:
K... leaflet is gone entirely now.